www/main/quick-tools-utils.js

```diff
@@ -0,0 +1,16 @@
+const firstCharToUpperCase = str => `${str.charAt(0).toUpperCase()}${str.slice(1)}`;
+
+export const changeFontSize = (iconType, increase = true) => {
+  const setterAction = `set${firstCharToUpperCase(iconType)}FontSize`;
+  const getterVar = `${iconType}FontSize`;
+  const oldValue = parseInt(global.getUserSettings[getterVar], 10);
+  const newValue = increase ? oldValue + 1 : oldValue - 1;
+  global.setUserSettings[setterAction](newValue);
```

Comment:
anyway to do this in a more functional approach? modifying globals can have pitfalls.

Reply:
Yes,

I have removed the need of having global.getUserSettings (using local storage) but for set I am still using global.setUserSettings - this is technically not modifying global. In either case this is a temporary hack, as we move to react everywhere, all of this (along with existing globals) will be removed.